modules/ppcp-applepay/resources/js/ApplepayButton.js

```diff
@@ -179,13 +186,15 @@ class ApplepayButton {
             appleContainer.innerHTML = `<apple-pay-button id="${id}" buttonstyle="${color}" type="${type}" locale="${language}">`;
         }
 
-        jQuery('#' + wrapper).addClass('ppcp-button-' + ppcpStyle.shape);
+        const $wrapper = jQuery('#' + wrapper);
+        $wrapper.addClass('ppcp-button-' + ppcpStyle.shape);
 
         if (ppcpStyle.height) {
-            jQuery('#' + wrapper).css('--apple-pay-button-height', `${ppcpStyle.height}px`)
+            $wrapper.css('--apple-pay-button-height', `${ppcpStyle.height}px`)
+            $wrapper.css('height', `${ppcpStyle.height}px`)
         }
 
-        jQuery(wrapper).append(appleContainer);
+        jQuery(wrapper).append(appleContainer); // ToDo: this selector doesnt seem valid.
```

Comment:
Yes, it seems to be doing nothing. `<apple-pay-button>` is already added because we set `innerHTML` above.